python/cac_tripplanner/templates/directions.html

```diff
@@ -1,10 +1,12 @@
 {% extends "base.html" %}
 {% block content %}
-{% include "header-mixin.html" %}
-<section class="directions">
-  <div class="directions-list">
-  </div>
-</section>
+<div class="print container">
+  {% include "header-mixin.html" %}
+  <section class="directions-print">
```

Comment:
Not having the `directions` class here will likely break some of the code that expects it, which may be why the `osm:node` labels are showing up in the screenshot (those should refresh to street names).